roles/hostedbm/tasks/post-deploy-step-on-guest-cluster.yml

```diff
@@ -0,0 +1,115 @@
+---
+- name: "deploy-operators : Install Metallb Operator on the Guest Cluster"
+  ansible.builtin.include_role:
+    name: olm_operator
```

Comment:
To use the role from the collection please use the FQCN: redhatci.ocp.olm_operator

Reply:
done